src/docbuilder/rustwide_builder.rs

```diff
@@ -122,8 +122,9 @@ impl RustwideBuilder {
 
         let mut targets_to_install = TARGETS
             .iter()
-            .map(|&t| t.to_string())
+            .map(|t| (*t).to_string()) // &str has a specialized ToString impl, while &&str goes through Display
             .collect::<HashSet<_>>();
```

Comment:
I liked this better before. The comment is fine but I prefer `|&t|` to the parentheses.